src/controllers/agproxysecret/secret.go

```diff
@@ -0,0 +1,126 @@
+package agproxysecret
```

Comment:
agproxysecret does not contain an actual controller, but is called from the Dynakube controller. I'd suggest moving the package either one up, i.e. `/src/agproxysecret`, or into the activegate package, i.e. `/src/controllers/activegate/agproxysecret`

Reply:
Moved to /src/agproxysecret